js folder/index.js

```diff
@@ -0,0 +1,65 @@
+// ---------------------- VARIABLES -------------------------- //
+
+const addBtn = document.querySelector('#addBtn');
+const title = document.querySelector('#titleId');
+const author = document.querySelector('#authorId');
+const bookUl = document.querySelector('#books');
+
+// ------------------------ DATA ----------------------------- //
+
+let booksData = [];
+
+// ------------------------ FUNCTIONS ------------------------ //
+function BookConstructor(title, author) {
+  this.title = title;
+  this.author = author;
+}
+
+function loadBooks(index) {
+  // creatingElements
+  const bookLi = document.createElement('li');
+  const titleP = document.createElement('p');
+  const authorP = document.createElement('p');
+  const deleteBtn = document.createElement('button');
+  deleteBtn.setAttribute('id', index);
+  deleteBtn.setAttribute('class', 'buttons');
+  // content inside
+  titleP.textContent = booksData[index].title;
+  authorP.textContent = booksData[index].author;
+  deleteBtn.textContent = 'remove';
+  // appendChilds
+  bookLi.appendChild(titleP);
+  bookLi.appendChild(authorP);
+  bookLi.appendChild(deleteBtn);
+  bookUl.appendChild(bookLi);
+
+  deleteBtn.addEventListener('click', () => {
+    booksData.splice(index, 1);
+    bookUl.removeChild(bookLi);
+    localStorage.setItem('books', JSON.stringify(booksData));
+  });
+}
+
+// ----------------------- EVENT LISTENER ------------------- //
+
+addBtn.addEventListener('click', () => {
+  const titleName = title.value;
+  const authorName = author.value;
+  title.value = '';
+  author.value = '';
+  const addNewBook = new BookConstructor(titleName, authorName);
+  booksData.push(addNewBook);
```

Comment:
[OPTIONAL] Consider using methods that don't mutate the array.

<img width="411" alt="Screen Shot 2022-06-27 at 17 45 14" src="https://user-images.githubusercontent.com/82426682/176049116-daee24c8-6246-4b34-bc00-4d5ce6fe3695.png">

Reply:
Ok, I'll give it a try. Could you explain why it can be a better idea?